completions/tox

```diff
@@ -0,0 +1,26 @@
+# tox(1) completion                                   -*- shell-script -*-
+
+_tox()
+{
+    local cur prev words cword
+    _init_completion || return
+
+    case $prev in
+        -h|--help|--version)
+            return
+            ;;
+        -e)
+            local envs=`tox --listenvs-all`
+            COMPREPLY=( $(compgen -W "${envs}" -- ${cur}) )
```

Comment:
Add spaces around `$( ... )`, and quote $cur, and remove superfluous curly braces from envs and cur.